azurelinuxagent/ga/exthandlers.py

```diff
@@ -715,17 +715,6 @@ def handle_enable(self, ext_handler_i, extension):
             ext_handler_i.ensure_consistent_data_for_mc()
             ext_handler_i.update_settings(extension)
 
-        # Always create a placeholder file for enable/disable command if not exists
-        # We don't create a placeholder for other commands because we use status file as a way to report Handler level
-        # failures back to CRP. If a placeholder for an extension already exists with Transitioning status, we would
-        # not override it, hence we only create a placeholder for enable/disable commands but the extensions have the
-        # data to create their own if needed.
-
-        # Note: Due to a bug in multiple extensions, we're only creating a default placeholder for Multi-Config extensions.
-        # A fix will follow soon where we will report transitioning status for extensions by default if no status file
-        # found instead of reporting an error.
-        if ext_handler_i.should_perform_multi_config_op(extension):
-            ext_handler_i.create_placeholder_status_file(extension)
```

Comment:
I see that we still have 3 calls to create_placeholder_status_file to create the status file on some error conditions.

Should we rename this function to avoid confusion with the placeholder file mentioned in the publisher's spec, and also remove the default values for the parameters, which seem to be targeted to this one call that you are removing.

Reply:
sure let me send an update